pipeline-runner/init.R

```diff
@@ -14,14 +14,50 @@ options(keep.source.pkgs = TRUE)
 for (f in list.files('R', '.R$', full.names = TRUE)) source(f, keep.source = TRUE)
 load('R/sysdata.rda') # constants
 
+buildActivityArn <- function(aws_region, aws_account_id, activity_id) {
+    if(is.na(activity_id)) {
+        return(NA)
+    }
+
+    activity_arn <- sprintf("arn:aws:states:%s:%s:activity:%s", aws_region, aws_account_id, activity_id)
+    message("Built activity ARN: ", activity_arn)
+    return(activity_arn)
+}
+
 load_config <- function(development_aws_server) {
+    label_path <- "/etc/podinfo/labels"
+    aws_account_id <- Sys.getenv("AWS_ACCOUNT_ID", unset="242905224710")
+    aws_region <- Sys.getenv("AWS_DEFAULT_REGION", unset="eu-west-1")
+
+    activity_arn <- NA
+
+    repeat {
+        if(file.exists(label_path)) {
```

Comment:
Is this running if we are in localhost? And if that is so, do we want it to run?

Reply:
yes, it runs in local, it will not find the file so it will proceed to next if and get the activityArn from the env variables.

Reply:
So this is used to check whether we are running in local or not? could you add a comment making that more explicit?

Reply:
done